src/common/Label/index.tsx

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import { Text, TextProps } from 'react-native';
+
+import styles from './styles';
+
+const Label: React.FunctionComponent<TextProps> = ({ children, style, ...props }) => {
+  return (
+    <Text style={[styles.label, style]} {...props}>
+      {children}
+    </Text>
+  );
+};
```

Comment:
```suggestion
const Label: React.FunctionComponent<TextProps> = ({ children, style, ...props }) => (
    <Text style={[styles.label, style]} {...props}>
      {children}
    </Text>
  );
```

If you want we generally use implicit return